utils/set/set_test.go

```diff
@@ -26,7 +26,7 @@ func TestSet(t *testing.T) {
 	s.Add(id1)
 	require.True(s.Contains(id1))
 	require.Len(s.List(), 1)
-	require.Equal(len(s.List()), 1)
+	require.Len(s.List(), 1)
```

Comment:
This line is a duplicate of the above line now - probably a bug with some prior refactor